Source/WebCore/platform/ScrollbarsController.h

```diff
@@ -95,6 +95,7 @@ class ScrollbarsController {
 
     WEBCORE_EXPORT virtual bool shouldDrawIntoScrollbarLayer(Scrollbar&) const { return true; }
     WEBCORE_EXPORT virtual bool shouldRegisterScrollbars() const { return true; }
+    WEBCORE_EXPORT virtual void setScrollbarEnabledSate(WebCore::Scrollbar&) { }
```

Comment:
No need for `WebCore::`

Reply:
It's odd to have a function called `setScrollbarEnabledSate` which doesn't take the state as an argument. Should this be called `updateScrollbarEnabledState()`?

Reply:
Fixed.